### DISCUSSION TITLE
Name Only Sender Address Format Option

### DISCUSSION THREAD

**Discussion by lv6vyzts93foqih3** [simple-login/app#525] [Feature Requests]
Add a third option to "Sender Address Format" setting that omits the original sender address entirely and just keeps the name. Some users (like myself) will prefer this option because you end up with a cleaner looking inbox (just sender names, no formatted addresses).

**Comment by lv6vyzts93foqih3** [comment]
#420 would also make this possible

**Comment by jumpingbrowntrout** [comment]
I have plus one'd this feature request but agree #420 would make this possible and i believe it is on the roadmap.
